ClinGen/clincoded
# Connects to #671. Additional changes to VCI UI framework.

This PR consists of additional changes pertinent to PR #688.

**Technical notes:**
1. The 4th item of **Known issues** in PR #688, in which the active interpretation is not highlighted/flagged in the header, has now been addressed.
2. The 5th item of **Known issues** in PR #688, in which clicking the "Edit" links to interpretations does not show the form elements in the "Population" tab even when the user did not start a new interpretation, has now been addressed.
3. The form component that was previously embedded in the population tab has now been moved to a file of its own so that it becomes reusable for other tabs.

**Other known issues:**
1. After clicking on the "Start new interpretation" button above the tabs, or clicking on the "Edit" link of a given interpretation, the "Associate with Disease" button on the right and the "Return to Evidence Only" button on the left will be shown above the tabs. However, the functionality of these 2 buttons have not been clearly identified and thus they are not functional.
2. The (met) criteria bar above the tabs has not been implemented.
3. Individual tab content are still in progress or yet to be implemented.

**Steps to test this PR:**
1. Login and go to http://localhost:6543/variant-central/?variant=35eedfdc-2685-11e5-b6a0-60f81dc5b05a
2. Verify that no interpretations in the header are either flagged or highlighted, and there are no form elements in the "Population" tab.
3. Click on the "Edit" link to the interpretation in the header, and expect to see a checkmark at the end of the interpretation flagging it as the active interpretation.
4. Then click on "Population" tab, and expect to see form elements being displayed in the "Population" tab.
